app/views/challenges/_resources.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="usa-card" data-controller="challenge-detail">
+  <div class="grid-row border-bottom" style="border-color: #c6cace">
+    <div class="grid-col">
+      <h2 class="font-sans-xl margin-y-2">Additional resources</h2>
+    </div>
+  </div>
+
+  <div class="margin-top-6">
+    <div class="usa-card usa-card__container">
+      <div class="usa-card-body">
+        <p>No additional resources available for this challenge.</p>
+      </div>
+    </div>
+  </div>  
```

Comment:
In theory this should be able to conditionally render a list of `@challenge.supporting_documents`, but I tried it on the Phoenix side and I can't seem to upload any supporting_documents to challenges even on staging.
maybe the best thing for now is to just hide the `Resources` tab menu item with a comment that the documents are non-functional right now. I don't know but this could be a vestigial feature.